sss/process_foms.c

```diff
@@ -380,32 +381,47 @@ static int ss_process_stats(struct m0_reqh           *reqh,
 	return M0_RC(rc);
 }
 
-static int ss_bytecount_stats_ingest(struct m0_ss_process_rep *rep)
+static int ss_bytecount_stats_ingest(struct m0_cob_domain *cdom,
+				     struct m0_ss_process_rep *rep)
 {
-	struct m0_cob_bckey  key;
-	struct m0_cob_bcrec  rec;
+	int                  rc;
+	struct m0_buf       *key_buf = NULL;
+	struct m0_buf       *rec_buf = NULL;
 
 	/* DUMMY values */
-	key.cbk_pfid = M0_FID_TINIT('v', 1, 8);
-	key.cbk_user_id = 8881212;
-
-	rec.cbr_bytecount = 10240000;
-	rec.cbr_cob_objects = 10000;
+	if (M0_FI_ENABLED("dummy_bytecount_data")) {
+		struct m0_cob_bckey  key;
+		struct m0_cob_bcrec  rec;
+
+		key.cbk_pfid = M0_FID_TINIT('v', 1, 8);
+		key.cbk_user_id = 8881212;
+		rec.cbr_bytecount = 10240000;
+		rec.cbr_cob_objects = 10000;
+		m0_buf_new_aligned(&rep->sspr_bckey, &key, sizeof(key), 0);
+		m0_buf_new_aligned(&rep->sspr_bcrec, &rec, sizeof(rec), 0);
+
+		return 0;
+	}
 
-	m0_buf_new_aligned(&rep->sspr_bckey, &key, sizeof(key), 0);
-	m0_buf_new_aligned(&rep->sspr_bcrec, &rec, sizeof(rec), 0);
+	rc = m0_cob_bc_entries_dump(cdom, &key_buf, &rec_buf, &rep->sspr_kv_count);
+	if (rc == 0) {
+		m0_buf_init(&rep->sspr_bckey, key_buf->b_addr, key_buf->b_nob);
+		m0_buf_init(&rep->sspr_bcrec, rec_buf->b_addr, rec_buf->b_nob);
+	}
```

Comment:
Check where these buffers are getting freed.

Reply:
It is getting freed in m0_ss_process_rep_fop_release(). This change was added in #1446
sss/process_fops.h